docs/whos-using.md

```diff
@@ -1,4 +1,4 @@
-# Who's using sequelize?
```

Comment:
I personally didn't like the removal of this question mark... Was this automatically suggested by `markdownlint`?

Reply:
Yes. It's a [rule](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md026) `Trailing punctuation in heading`.

Reply:
But I prefer to follow the rule. No question mark is more like a title.

Reply:
I see... I would personally suggest changing the default config of markdownlint to allow question marks. It's even an example in the docs you linked to. But if you really disagree, at least I think the whole title should be changed then. "Who's using sequelize" without a question mark seems odd to me. Would you suggest a different title?

Reply:
Um..... I was persuaded. Add the question mark back.